content/blog/2017/07/2017-07-22-powershell-pipeline.adoc

```diff
@@ -0,0 +1,198 @@
+---
+layout: post
+title: "Microsoft PowerShell Support for Jenkins Pipelines"
+tags:
+- durable task
+- powershell
+author: gabloe
+---
+
+I am pleased to announce Microsoft PowerShell support for Jenkins Pipelines.
+With this addition you will now be able to run Microsoft PowerShell scripts directly in your Jenkins Pipeline 
+projects.
+This blog post covers the basics of getting started with Microsoft PowerShell in your pipeline
+and provides some basic examples.
+
+== Introduction to Microsoft PowerShell
+PowerShell is Microsoft's open source and cross platform command line shell,  as well as an 
+automation and configuration tool/framework which has a broad user base. PowerShell can be used to perform common system administration tasks in Windows, macOS, and Linux environments, as well as be used
+as a general purpose scripting language. Now that Jenkins Pipelines has
+support for PowerShell, you can enjoy the rich set of features in PowerShell for your
+daily devops work.
+
+Before diving into using PowerShell in your pipeline, I recommend reading the 
+link:https://msdn.microsoft.com/en-us/library/ms714469(v=vs.85).aspx[Windows PowerShell Reference]
+as well as the link:https://blogs.msdn.microsoft.com/powershell/[PowerShell Team Blog] for an introduction
+to PowerShell features and utilities, as well as a quick look into what makes up the language of PowerShell. Microsoft also 
+has an active link:https://github.com/powershell/powershell[PowerShell community] on GitHub, which I highly recommend
+visiting and submitting feature requests and bug reports as you see fit. Jenkins Pipelines currently supports Microsoft 
+PowerShell 3.0 or higher, so also be sure to check which version of PowerShell is installed on your system so that you know which features you can take advantage of in your pipeline. Please note that we recommend that you upgrade to the latest stable version of PowerShell available, which as of this writing is version 5.1.14393.
```

Comment:
I would recommend putting a very basic simple version of the powershell step at this point in the blog post, e.g. 

```groovy
node {
  powershell 'Write-Output "Hello World"'
}
```